packages/dashboard/src/pages/Settings/components/BackupPreviewCard.tsx

```diff
@@ -0,0 +1,82 @@
+import { MaskColorVar } from '@dimensiondev/maskbook-theme'
+import { makeStyles } from '@material-ui/styles'
+import classNames from 'classnames'
+
+const useStyles = makeStyles(() => ({
+    root: {
+        padding: '19px 24px 9px',
+        minHeight: 205,
+        borderRadius: 8,
+        background: MaskColorVar.infoBackground,
+    },
+    item: {
+        paddingBottom: 10,
+        display: 'flex',
+        justifyContent: 'space-between',
+    },
+    sub: {
+        paddingLeft: 60,
+    },
+}))
+
+interface Props {
+    json: {
+        personas: { linkedProfiles: [] }[]
+        posts: {}[]
+        profiles: {}[]
+        wallets: {}[]
+    }
```

Comment:
Should add a new type for the UI displaying of the backup file. Because the backup file has different versions and might have more details than needed. Convert/summary the backup to the UI type should be done in the background page.

Reply:
updated